modules/fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/fs2/io/Metrics.scala

```diff
@@ -47,10 +49,10 @@ trait Metrics[F[_]] {
 object Metrics {
 
   val LoggerName = "enrich.metrics"
-  val LatencyGaugeName = "enrich.metrics.latency"
-  val RawCounterName = "enrich.metrics.raw.count"
-  val GoodCounterName = "enrich.metrics.good.count"
-  val BadCounterName = "enrich.metrics.bad.count"
+  val LatencyGaugeName = "snowplow.enrich.metrics.latency"
+  val RawCounterName = "snowplow.enrich.metrics.raw"
+  val GoodCounterName = "snowplow.enrich.metrics.good"
+  val BadCounterName = "snowplow.enrich.metrics.bad"
```

Comment:
I am slightly worried it is a breaking change to rename these metrics.  On the other hand, fs2 is currently only released as "public beta", and until now metrics have been used only for debugging.  I think the new names are more appropriate when we start sending metrics to a statsd server.